examples/cra-todo-list-auth/.gitignore

```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
```

Comment:
Maybe add the .idea folder here to make it slightly simpler, too?

Reply:
Will remove it 👍